bower.json

```diff
@@ -22,7 +22,10 @@
     "iron-icon": "PolymerElements/iron-icon#^2.0.0",
     "paper-button": "PolymerElements/paper-button#^2.0.0",
     "iron-lazy-pages": "TimvdLippe/iron-lazy-pages#^2.0.2",
-    "firebase": "^4.1.3"
+    "firebase": "^4.1.3",
+    "marked-element": "PolymerElements/marked-element#^2.1.1",
+    "clamp-js": "*",
```

Comment:
Any version. It's old library, so using what we have
https://libraries.io/bower/clamp-js